includes/wikia/Wikia.php

```diff
@@ -1252,8 +1252,14 @@ static public function onWebRequestInitialized( WebRequest $request ) {
 	 * @return bool true, it's a hook
 	 */
 	static public function outputHTTPSHeaders( WebRequest $request ) {
+		global $wgDevelEnvironment, $wgDevDomain;
 		if ( WebRequest::detectProtocol() === 'https' ) {
-			$request->response()->header( "Content-Security-Policy-Report-Only: default-src https:; script-src https: 'unsafe-inline' 'unsafe-eval'; style-src https: 'unsafe-inline'; img-src https: data:; report-uri " . self::CSP_ENDPOINT );
+			if ( !empty( $wgDevelEnvironment ) ) {
```

Comment:
Cannot we simply use `"services.{$wgWikiaBaseDomain}"` here?

Reply:
`wgWikiaBaseDomain` is always `wikia.com` even on devboxes. We could make a services domain config var that all code can use though, and move this to config. What do you think?

Reply:
Yes, I would prefer having services domain in config. Can be similar to how `wgDiscussionsApiUrl` is implemented.

Reply:
And done: https://github.com/Wikia/config/pull/2385